providers/fab/src/airflow/providers/fab/www/package.json

```diff
@@ -46,7 +46,7 @@
     "babel-jest": "^27.3.1",
     "babel-loader": "^9.1.0",
     "clean-webpack-plugin": "^3.0.0",
-    "copy-webpack-plugin": "^12.0.2",
+    "copy-webpack-plugin": "^6.0.3",
```

Comment:
When I copied over the Flask application from core Airflow to FAB provider, I tried to upgrade some dependencies along the way. `copy-webpack-plugin` was one of them. But upgrading it causes some problems, the property `flatten` used in the webpack config is deleted from this version. I think this could be fixed but we can do it later in a separate PR. I reverted and set back the version we were using in core Airflow